src/tools/rustfmt/atom.md

```diff
@@ -1,8 +1,8 @@
 # Running Rustfmt from Atom
 
-## RLS
+## rust-analyzer
 
-Rustfmt is included with the Rust Language Server, itself provided by [ide-rust](https://atom.io/packages/ide-rust).
+Rustfmt is included with rust-analyzer, itself provided by [ide-rust](https://atom.io/packages/ide-rust).
```

Comment:
Is this true today? I think rust-analyzer shells out to rustfmt, doesn't directly inline it (otherwise can't build on stable)?

I would expect users to need to rustup component add rustfmt or so.

Reply:
Good point, and yes that's correct as per my understanding as well. RLS was a rustfmt-as-a-lib consumer, but RA invokes rustfmt directly using the stdin mode to pipe in file/snippet contents.

I'm not sure how much utility these docs provide TBH as they've not really been kept up to date, but if we want to make an update perhaps something to the effect of:

```markdown
Rustfmt can be utilized from rust-analyzer,...
```

Reply:
Good catch!  I was not reading it closely. I have tweaked the wording so it does not imply that rust-analyzer installs rustfmt.